src/coprocessor/endpoint.rs

```diff
@@ -1726,92 +1763,116 @@ mod tests {
             let resp = &rx.recv().unwrap()[0];
             assert!(resp.get_other_error().is_empty());
             assert_ge!(
-                resp.get_exec_details()
-                    .get_time_detail()
-                    .get_process_wall_time_ms(),
-                PAYLOAD_LARGE.saturating_sub(COARSE_ERROR_MS)
+                Duration::from_nanos(
```

Comment:
Also add test for v1? make sure it's currently compatible.

Reply:
done. Add a check that the values in TimeDetail and TimeDetailV2 are equal.